yaml_specs/modulemd_stream_v2.yaml

```diff
@@ -381,24 +608,50 @@ data:
             - xxx-0:1-1.module_deadbeef.x86_64
             - xxx-0:1-1.module_deadbeef.i686
             - xyz-0:1-1.module_deadbeef.x86_64
+
         # The rpm-map exists to link checksums from repomd to specific
```

Comment:
Add `# rpm-map:` here, and add other hard-coded keys throughout the rest of this block.